<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgChartDlg</class>
 <widget class="QDialog" name="CfgChartDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>531</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,3,5" columnminimumwidth="0,0,0,0">
    <item row="3" column="3">
     <widget class="QCheckBox" name="checkUpMacd">
      <property name="text">
       <string>显示MACD</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2" rowspan="2">
     <widget class="QSpinBox" name="spinDownMacdPercent"/>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="checkDownK">
      <property name="text">
       <string>显示K线图</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioUpVol">
        <property name="text">
         <string>交易量</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioUpMacd2">
        <property name="text">
         <string>MACD2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioUp2Hide">
        <property name="text">
         <string>不显示</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioDownVol">
        <property name="text">
         <string>交易量</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDownMacd2">
        <property name="text">
         <string>MACD2</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDown2Hide">
        <property name="text">
         <string>不显示</string>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>K线图</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>K线图</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QSpinBox" name="spinUpMacdFast"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinUpMacdSlow"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinUpMacdDea"/>
      </item>
     </layout>
    </item>
    <item row="9" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QSpinBox" name="spinDownMacdFast"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinDownMacdSlow"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinDownMacdDea"/>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinUpKPercent"/>
    </item>
    <item row="8" column="3">
     <widget class="QCheckBox" name="checkDownMacd">
      <property name="text">
       <string>显示MACD</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="checkUpK">
      <property name="text">
       <string>显示K线图</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QSpinBox" name="spinUpMacd2Fast"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinUpMacd2Slow"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinUpMacd2Dea"/>
      </item>
     </layout>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="spinDownKPercent"/>
    </item>
    <item row="7" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QSpinBox" name="spinDownMacd2Fast"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinDownMacd2Slow"/>
      </item>
      <item>
       <widget class="QSpinBox" name="spinDownMacd2Dea"/>
      </item>
     </layout>
    </item>
    <item row="6" column="2" rowspan="2">
     <widget class="QSpinBox" name="spinDownVolPercent"/>
    </item>
    <item row="1" column="2" rowspan="2">
     <widget class="QSpinBox" name="spinUpVolPercent"/>
    </item>
    <item row="3" column="2" rowspan="2">
     <widget class="QSpinBox" name="spinUpMacdPercent"/>
    </item>
    <item row="1" column="1" rowspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>交易量或MACD2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>MACD</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" rowspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>交易量或MACD2</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" rowspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>MACD</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="5">
     <widget class="QCheckBox" name="checkDown">
      <property name="text">
       <string>下图</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QCheckBox" name="checkUp">
      <property name="text">
       <string>上图</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonTest">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>试用</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CfgChartDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CfgChartDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
